test/osde2e/managed_api_test.go

```diff
@@ -21,47 +21,26 @@ var _ = Describe("integreatly", func() {
 	})
 
 	RunTests := func() {
+		watchNs := os.Getenv("WATCH_NAMESPACE")
 
-		os.Setenv("WATCH_NAMESPACE", "redhat-rhoam-operator")
-		os.Setenv("SKIP_FLAKES", "true")
-		common.NamespacePrefix = "redhat-rhoam-"
-		common.RHMIOperatorNamespace = common.NamespacePrefix + "operator"
-		common.MonitoringOperatorNamespace = common.NamespacePrefix + "middleware-monitoring-operator"
-		common.MonitoringFederateNamespace = common.NamespacePrefix + "middleware-monitoring-federate"
-		common.AMQOnlineOperatorNamespace = common.NamespacePrefix + "amq-online"
-		common.ApicurioRegistryProductNamespace = common.NamespacePrefix + "apicurio-registry"
-		common.ApicurioRegistryOperatorNamespace = common.ApicurioRegistryProductNamespace + "-operator"
-		common.ApicuritoProductNamespace = common.NamespacePrefix + "apicurito"
-		common.ApicuritoOperatorNamespace = common.ApicuritoProductNamespace + "-operator"
-		common.CloudResourceOperatorNamespace = common.NamespacePrefix + "cloud-resources-operator"
-		common.CodeReadyProductNamespace = common.NamespacePrefix + "codeready-workspaces"
-		common.CodeReadyOperatorNamespace = common.CodeReadyProductNamespace + "-operator"
-		common.FuseProductNamespace = common.NamespacePrefix + "fuse"
-		common.FuseOperatorNamespace = common.FuseProductNamespace + "-operator"
-		common.RHSSOUserProductOperatorNamespace = common.NamespacePrefix + "user-sso"
-		common.RHSSOUserOperatorNamespace = common.RHSSOUserProductOperatorNamespace + "-operator"
-		common.RHSSOProductNamespace = common.NamespacePrefix + "rhsso"
-		common.RHSSOOperatorNamespace = common.RHSSOProductNamespace + "-operator"
-		common.SolutionExplorerProductNamespace = common.NamespacePrefix + "solution-explorer"
-		common.SolutionExplorerOperatorNamespace = common.SolutionExplorerProductNamespace + "-operator"
-		common.ThreeScaleProductNamespace = common.NamespacePrefix + "3scale"
-		common.ThreeScaleOperatorNamespace = common.ThreeScaleProductNamespace + "-operator"
-		common.UPSProductNamespace = common.NamespacePrefix + "ups"
-		common.UPSOperatorNamespace = common.UPSProductNamespace + "-operator"
-		common.MonitoringSpecNamespace = common.NamespacePrefix + "monitoring"
-		common.Marin3rOperatorNamespace = common.NamespacePrefix + "marin3r-operator"
-		common.Marin3rProductNamespace = common.NamespacePrefix + "marin3r"
-		common.CustomerGrafanaNamespace = common.NamespacePrefix + "customer-monitoring-operator"
+		// Assign the installation name by WATCH_NAMESPACE - if no installation name has been found fail the test.
+		if watchNs == "redhat-rhmi-operator" {
+			installType = "managed"
+		} else if watchNs == "redhat-rhoam-operator" {
+			installType = "managed-api"
+		} else {
+			t.Fatalf("No installation name has been found")
+		}
```

Comment:
I wonder would this be more readable as a switch statement. 
```suggestion
               switch watchNs {
                  case "redhat-rhmi-operator":
                     installType = "managed"
                  case "redhat-rhoam-operator":
                     installType = "managed-api"   
                  default:
                     t.Fatalf("No installation name has been found")
                   }
```
I am not to pushed on this and I know may indents may not be correct in the example.

Reply:
hmm, now that I think of it...we actually don't need this check. The `installType` is pulled from suite_test.go file here: https://github.com/integr8ly/integreatly-operator/blob/dbd8ceda04ad3123155186078f860f1ca1beeaaf/test/osde2e/suite_test.go#L35, then it gets a value and only then the managed-api-test is triggered. I think we should be ok with removing the entire logic around reading WATCH_NAMESPACE here but need to verify it first!

Reply:
Just verified it, it works without reading the WATCH_NAMESPACE in the managed-api-test.go and looks much cleaner!